New google cloud broker (#106)
kyma-project/addons: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/overview.md

---
title: Overview
type: Overview
---

>**NOTE:** To provision this class, first you must create a Secret. Read the following document to learn how.

The Google Cloud Platform Service Broker class exposes services from the [Google Cloud Platform Service Broker](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker)
in a given Namespace of the Kyma cluster.

> The embedded MySQL DB is used to keep the broker's state so uninstalling the broker will delete all information about created instances but created services will still exists.

Reviewer: ```suggestion
The embedded MySQL database is used to keep the broker's state. Uninstalling the broker will delete all information about created instances but created services will still exist.
```

## Create a Secret

> To add the Google Cloud Platform Service Broker to your Namespace, you must prepare a service account and a 

Reviewer: ```suggestion
To add the Google Cloud Platform Service Broker to your Namespace, prepare a service account and a 
```

JSON access key. 

Follow these steps to create a Kubernetes Secret which contains a JSON access key:

> 1. Open https://console.cloud.google.com/ and select your project.

Reviewer: ```suggestion
1. Open `https://console.cloud.google.com/` and select your project.
```
